app/assets/javascripts/application.js

```diff
@@ -45,6 +46,11 @@
 // Jtruncate plugin, http://www.jeremymartin.name/projects.php?project=jTruncate
 // modified by Scott Miller- remove animation, newline for more link
 
+$(document).ready(function() {
+  /* Activating Best In Place */
+  jQuery(".best_in_place").best_in_place();
+});
+
```

Comment:
You will need to move this into `app.js` inside the `Helpy.ready()` function so that it works across ajax page change events.  You'll notice that currently it does not work correctly if you click around several times to different tickets and views in the admin.